test/utils/TestConstants.scala

```diff
@@ -24,6 +24,7 @@ object TestConstants {
 
   // We only want one test nino and utr throughout, therefore assign a value in the object declaration
   lazy val testUtr = new SaUtrGenerator().nextSaUtr.utr
+  lazy val testNino = "AB654321B"
```

Comment:
Use a nino generator instead of a fixed Nino as this could be considered PII and shouldn't be in a repo
```
  import uk.gov.hmrc.domain.Generator
  
  lazy val testNino = new Generator().nextNino
```

Reply:
;+1: Thanks I was looking for this.